schedule/index.html

```diff
@@ -120,12 +123,17 @@ <h2 class="track-title">$track-title</h2>
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" 
-      integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" 
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"
+      integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS"
       crossorigin="anonymous"></script>
     <script type="text/javascript">
       var ts = new Date().valueOf() // cache breaker
       document.write('<script src="./main.js?t=' + ts + '"><\/script>')
     </script>
+    <script language="javascript" type="text/javascript">
+        $(window).load(function() {
+            $('#loading').hide();
```

Comment:
I'm not sure what this is doing, but it should be hooked in to the SessionStores 'data' event (see `main.js`, ca. line 257) - i.e. loader should display until the session data was loaded via ajax.